packages/ui/src/theme/tokens/components/copy.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  fontSize: { value: '{font.sizes.xs.value}' },
+  gap: { value: '{space.relative.medium.value}' },
+
+  svg: {
+    path: { fill: { value: '{space.relative.medium.value}' } },
+  },
+  toolTip: {
+    bottom: { value: '{space.large}' },
```

Comment:
This is missing `.value` at the end of the reference.

Reply:
I experimented both and found it works with out `value`, so I stopped adding it. Do we have a preference?

Reply:
Oh yea I added that feature in style dictionary recently... never mind. We could drop `.value` everywhere in our tokens.